content/docs/200-inputs/400-select.mdx

```diff
@@ -7,9 +7,30 @@ Add a _select_ input to your app to allow users to choose from a list of many op
 
 ## Options
 
-Add options to your _select_ input and set a default option. The input ID in the URL template for your _select_ input is replaced with the selected option’s value.
-
-<Callout>**Upcoming feature:** Fetch options from a remote source.</Callout>
+Select options can be added manually or pulled from a remote source.
+The input ID in the URL template for your _select_ input is replaced with the selected option’s value.
+
+### Manual Options
+
+Select the "manual" options type, then add options to your _select_ input and set a default option. 
+
+
+### Remote Options
+
+Select the "remote" options type, then specify the options URL and any required headers.
+The API must return a JSON response in the following format.
+```json
+[
+  {
+    "label": "my label",
+    "value": "my-value",
+  },
+  {
+    "label": "my second label",
+    "value": "my-second-value",
+  }
+]
+```
```

Comment:
Looks great! I think we should document how to set the default value in both the manual and remote sections. 

For remote options I believe you just need to return `{ label: '...', value: '...', default: true }`

Reply:
Good call! I managed to test this using https://mocki.io/ and it works perfectly! Even with multi-selects.

Updated here: https://github.com/mirainc/developer-docs/pull/46/commits/beb0508d628e520b034cf376caa36d0449e83361